src/messages/GE.json

```diff
@@ -280,5 +280,7 @@
   "running the query": "When you sign in, running the query will affect the data in your tenant.",
   "We did not find any history items": "We did not find any history items",
   "Expand response": "Expand response",
-  "Access Token": "Access token"
+  "Access Token": "Access token",
+  "Graph toolkit": "Graph toolkit",
+  "We did not find a Graph toolkit for this query": "We did not find a Graph toolkit for this query"
```

Comment:
Could this message be something like: `Woops, it looks like we don't have a Graph Toolkit sample for this query yet. Learn more about the Microsoft Graph Toolkit at https://aka.ms/mgt"

Reply:
Yeah.. It can

Reply:
Let's use this phrase recommended by Laura when there's no toolkit available.

No toolkit component is available for this response. To learn more about and contribute to the toolkit, see the [Microsoft Graph Toolkit public repo](https://github.com/microsoftgraph/microsoft-graph-toolkit)

@thewahome @nmetulev